File: commands/erin/bot-status.js
const Discord = require('discord.js');
const bot = require('../../config/bot.json');
const ee = require('../../config/embed.json');

module.exports = {
    name: 'bot-status',
    aliases: ['bot-update', 'botstatus', 'botupdate'],
    description: 'Pushes an embed to display in the channel about a certain update.',
    usage: '++bot-status Status Message',
    modOnly: 'yes',
    inHelp: 'yes',
    userPerms: [''],
    async execute(message, args, client) {

        const channel = client.channels.cache.find(channel => channel.id === bot.announcementsId);
        const reason = args.slice(0).join(" ");
        if (!reason) return message.reply('Mods, you forgot to include a status message. SMH');


        let embed = new Discord.MessageEmbed()
            .setColor(ee.bot_status)
            .setTitle('Hello, The Moderators have a new update for you!')
            .setDescription(`${reason}`)
            .setTimestamp()
            .setFooter('Want to suggest a feature for the bot? Use ++suggest');
        message.react('👍');
        channel.send({ content: `Hey, <@&772154227459883019>,`, embeds: [embed] }) // Subreddit Updates 780111997861363742 or Bot Updates 772154227459883019 or Server Updates 772153457111990282




    }
};

File: lib/src/commands/erin/bot_status.dart
import 'package:codinghelp_bot/codinghelp_bot.dart';
import 'package:codinghelp_bot/src/checks.dart';
import 'package:nyxx/nyxx.dart';
import 'package:nyxx_commands/nyxx_commands.dart';

final botStatus = ChatCommand.textOnly(
  'bot-status',
  'Pushes an embed to display in the channel about a certain update.',
  aliases: ['bot-update', 'botstatus', 'botupdate'],
  checks: [modCheck],
  id('bot-status', (
    MessageChatContext context,
    @UseConverter(greedyStringConverter) String status,
  ) async {
    final channel = context.client.channels[announcementChannelId] as ITextChannel;

    final embed = EmbedBuilder()
      ..color = DiscordColor.purple
      ..title = 'Hello, Erin has a new update for you!'
      ..description = status
      ..timestamp = DateTime.now()
      ..addFooter((footer) {
        footer.text = 'Want to suggest a feature for the bot? Use ++suggest';
      });

    await channel.sendMessage(
      MessageBuilder.content('Hey, <@&772154227459883019>,')..embeds = [embed],
    );

    await context.message.createReaction(UnicodeEmoji('👍'));
  }),
);
